### PR TITLE
TELCODOCS#1962: Release notes for new features in 4.17

### PR DESCRIPTION
**Pull request by sr1kar99**
Version(s):
4.17

Issue:
[TELCODOCS-1968](https://issues.redhat.com/browse/TELCODOCS-1968)
[TELCODOCS-1962](https://issues.redhat.com/browse/TELCODOCS-1962)

Link to docs preview:
- [Networking](https://82366--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-17-release-notes.html#ocp-4-17-networking_release-notes)
- [Edge computing](https://82366--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-17-release-notes.html#ocp-4-17-edge-computing_release-notes)
- [Edge computing Technology Preview features](https://82366--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-17-release-notes.html#ocp-4-17-technology-preview-tables_release-notes)

QE review:
- [x] QE has approved this change.

### CODE REVIEW

**Comment by sr1kar99** [comment]
@edcdavid 
Could you please review the release note for - [TELCODOCS-1841](https://issues.redhat.com/browse/TELCODOCS-1841)

**Comment by sr1kar99** [comment]
@sabinaaledort 
Could you please review the release note for - [TELCODOCS-1996](https://issues.redhat.com/browse/TELCODOCS-1996)
Thanks!

**Comment by sr1kar99** [comment]
@zeeke 
Could you please review the release note for - [TELCODOCS-1877](https://issues.redhat.com/browse/TELCODOCS-1877)
Thanks!

**Comment by yogeshahiray** [comment]
Approved from system testing side

**Comment by sabinaaledort** [comment]
> Could you please review the release note for - [TELCODOCS-1996](https://issues.redhat.com/browse/TELCODOCS-1996)

lgtm

**Comment by zeeke** [comment]
> @zeeke Could you please review the release note for - [TELCODOCS-1877](https://issues.redhat.com/browse/TELCODOCS-1877) Thanks!

lgtm!

**Comment by sr1kar99** [comment]
@ajaggapa 
Could you please review the release note for - [TELCODOCS-1877](https://issues.redhat.com/browse/TELCODOCS-1877)
Thanks!

**Comment by bradyallenjohnson** [comment]
The IPSec part looks good to me.

**Comment by ajaggapa** [comment]
lgtm 👍

File: release_notes/ocp-4-17-release-notes.adoc [hunk 1]
```diff
@@ -834,6 +851,21 @@ With this release, the image-based upgrade introduces the following enhancements
 
 For more information about the `ImageBasedGroupUpgrade` API, see xref:../edge_computing/image_based_upgrade/ztp-image-based-upgrade.adoc#ztp-image-based-upgrade-concept_ztp-gitops[Managing the image-based upgrade at scale using the ImageBasedGroupUpgrade CR on the hub].
 
+[id="ocp-4-17-disk-encryption_{context}"]
+==== Enabling disk encryption with TPM and PCR protection (Technology Preview)
```
Comment (sheriff-rh) [inline release_notes/ocp-4-17-release-notes.adoc:855]: I noticed this has the same title as the xref below. Considering this is more reference than procedure, maybe consider changing the title to "About disk encryption.." ?

File: release_notes/ocp-4-17-release-notes.adoc [hunk 2]
```diff
@@ -834,6 +851,21 @@ With this release, the image-based upgrade introduces the following enhancements
...
+
+With this release, you can enable disk encryption with Trusted Platform Module (TPM) and Platform Configuration Registers (PCRs) protection. You can use the `diskEncryption` field in the `SiteConfig` custom resource (CR) to configure the disk encryption. Configuring the `SiteConfig` CR enables disk encryption at the time of cluster installation.
+
+For more information, see xref:../edge_computing/ztp-reference-cluster-configuration-for-vdu.adoc#ztp-sno-du-configuring-disk-encryption-with-pcr-protection_sno-configure-for-vdu[Enabling disk encryption with TPM and PCR protection]
+
+[id="ocp-4-17-config-ipsec-encryption_{context}"]
+==== Configuring IPsec encryption for multi-node clusters using {ztp} and SiteConfig resources
```
Comment (sheriff-rh) [inline release_notes/ocp-4-17-release-notes.adoc:862]: Same here - consider re-titling to "About IPsec encryption...".

**Review by sheriff-rh — COMMENTED** [review]
Looks great - I would recommend some different titles for headers to avoid confusion.

File: release_notes/ocp-4-17-release-notes.adoc (hunk 1 above)
Comment (sr1kar99) [reply release_notes/ocp-4-17-release-notes.adoc:855]: Updated as per the suggestion.

File: release_notes/ocp-4-17-release-notes.adoc (hunk 2 above)
Comment (sr1kar99) [reply release_notes/ocp-4-17-release-notes.adoc:862]: Updated as per the suggestion.

File: release_notes/ocp-4-17-release-notes.adoc [hunk 3]
```diff
@@ -848,6 +865,21 @@ With this release, the image-based upgrade introduces the following enhancements
 
 For more information about the `ImageBasedGroupUpgrade` API, see xref:../edge_computing/image_based_upgrade/ztp-image-based-upgrade.adoc#ztp-image-based-upgrade-concept_ztp-gitops[Managing the image-based upgrade at scale using the ImageBasedGroupUpgrade CR on the hub].
 
+[id="ocp-4-17-disk-encryption_{context}"]
+==== About disk encryption with TPM and PCR protection (Technology Preview)
```
Comment (jeana-redhat) [inline release_notes/ocp-4-17-release-notes.adoc:869]:
```suggestion
==== Disk encryption with TPM and PCR protection (Technology Preview)
```

File: release_notes/ocp-4-17-release-notes.adoc [hunk 4]
```diff
@@ -848,6 +865,21 @@ With this release, the image-based upgrade introduces the following enhancements
...
+
+With this release, you can enable disk encryption with Trusted Platform Module (TPM) and Platform Configuration Registers (PCRs) protection. You can use the `diskEncryption` field in the `SiteConfig` custom resource (CR) to configure the disk encryption. Configuring the `SiteConfig` CR enables disk encryption at the time of cluster installation.
+
+For more information, see xref:../edge_computing/ztp-reference-cluster-configuration-for-vdu.adoc#ztp-sno-du-configuring-disk-encryption-with-pcr-protection_sno-configure-for-vdu[Enabling disk encryption with TPM and PCR protection]
+
+[id="ocp-4-17-config-ipsec-encryption_{context}"]
+==== About IPsec encryption for multi-node clusters using {ztp} and SiteConfig resources
```
Comment (jeana-redhat) [inline release_notes/ocp-4-17-release-notes.adoc:876]:
```suggestion
==== IPsec encryption for multi-node clusters using {ztp} and SiteConfig resources
```

File: release_notes/ocp-4-17-release-notes.adoc [hunk 5]
```diff
@@ -848,6 +865,21 @@ With this release, the image-based upgrade introduces the following enhancements
...
+
+You can now enable IPsec encryption in managed multi-node clusters that you deploy with GitOps ZTP and {rh-rhacm-first}. You can encrypt traffic between the managed cluster and IPsec endpoints external to the managed cluster. All network traffic between nodes on the OVN-Kubernetes cluster network is encrypted with IPsec in Transport mode.
```
Comment (jeana-redhat) [inline release_notes/ocp-4-17-release-notes.adoc:878]:
```suggestion
You can now enable IPsec encryption in managed multi-node clusters that you deploy with {ztp} and {rh-rhacm-first}. You can encrypt traffic between the managed cluster and IPsec endpoints external to the managed cluster. All network traffic between nodes on the OVN-Kubernetes cluster network is encrypted with IPsec in Transport mode. 
```

File: release_notes/ocp-4-17-release-notes.adoc [hunk 6]
```diff
@@ -848,6 +865,21 @@ With this release, the image-based upgrade introduces the following enhancements
...
+
+For more information, see xref:../edge_computing/ztp-reference-cluster-configuration-for-vdu.adoc#ztp-sno-du-configuring-disk-encryption-with-pcr-protection_sno-configure-for-vdu[Enabling disk encryption with TPM and PCR protection]
+
+[id="ocp-4-17-config-ipsec-encryption_{context}"]
+==== About IPsec encryption for multi-node clusters using {ztp} and SiteConfig resources
+
+You can now enable IPsec encryption in managed multi-node clusters that you deploy with GitOps ZTP and {rh-rhacm-first}. You can encrypt traffic between the managed cluster and IPsec endpoints external to the managed cluster. All network traffic between nodes on the OVN-Kubernetes cluster network is encrypted with IPsec in Transport mode. 
+
+For more information, see xref:../edge_computing/ztp-deploying-far-edge-sites.adoc#ztp-configuring-ipsec-using-ztp-and-siteconfig-for-mno_ztp-deploying-far-edge-sites[Configuring IPsec encryption for multi-node clusters using GitOps ZTP and SiteConfig resources]
```
Comment (jeana-redhat) [inline release_notes/ocp-4-17-release-notes.adoc:880]:
```suggestion
For more information, see xref:../edge_computing/ztp-deploying-far-edge-sites.adoc#ztp-configuring-ipsec-using-ztp-and-siteconfig-for-mno_ztp-deploying-far-edge-sites[Configuring IPsec encryption for multi-node clusters using {ztp} and SiteConfig resources]
```

**Review by jeana-redhat — COMMENTED** [review]
Just a couple nits here - mostly want to keep the heading format consistent by dropping the "About" in them.

Otherwise ready to merge.

/remove-label merge-review-in-progress
/remove-label merge-review-needed

File: release_notes/ocp-4-17-release-notes.adoc (hunk 3 above)
Comment (sr1kar99) [reply release_notes/ocp-4-17-release-notes.adoc:869]: Updated as per the suggestion.

File: release_notes/ocp-4-17-release-notes.adoc (hunk 4 above)
Comment (sr1kar99) [reply release_notes/ocp-4-17-release-notes.adoc:876]: Updated as per the suggestion.

File: release_notes/ocp-4-17-release-notes.adoc (hunk 5 above)
Comment (sr1kar99) [reply release_notes/ocp-4-17-release-notes.adoc:878]: Updated as per the suggestion.

File: release_notes/ocp-4-17-release-notes.adoc (hunk 6 above)
Comment (sr1kar99) [reply release_notes/ocp-4-17-release-notes.adoc]: Updated as per the suggestion.

**Pull request merged**